src/github/index.ts

```diff
@@ -5,14 +5,24 @@ import pullRequest from './pull-request';
 import push from './push';
 import { createBranch, deleteBranch } from './branch';
 import webhookTimeout from '../middleware/webhook-timeout';
+import bunyan from 'bunyan';
```

Comment:
We really should centralize this as our own `Logger` so we can abstract the implementation and change options globally.

Reply:
Ha. I actually had a centralized logger until I merged everything into your TS branch. Between my logger changes and your logger changes it got too messy so I ripped it out. I think that should be done as a separate PR, however.

Reply:
Agreed.